R/promise.R

```diff
@@ -68,12 +68,10 @@ Promise <- R6::R6Class(
       private$visible <- visible
       private$state <- "fulfilled"
 
-      later::later(function() {
```

Comment:
Please revert and add an expanded comment on why we shouldn't remove the `later` wrapper.

Reply:
I've also added some simple tests that define the current behaviour. I've added the comment that this isn't necessarily set in stone (I note that there are extensive existing tests covering the so-called javascript promises A+ specs, so perhaps this isn't mandated by those).